MastodonSDK/Sources/MastodonUI/Extension/MetaEntity+Accessibility.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  MetaEntity+Accessibility.swift
+//  
+//
+//  Created by Jed Fox on 2022-11-03.
+//
+
+import Meta
+
+extension Meta.Entity {
+    var accessibilityCustomActionLabel: String? {
+        switch meta {
+        case .url(_, trimmed: _, url: let url, userInfo: _):
+            return "Link: \(url)"
+        case .hashtag(_, hashtag: let hashtag, userInfo: _):
+            return "Hashtag \(hashtag)"
+        case .mention(_, mention: let mention, userInfo: _):
+            return "Show Profile: @\(mention)"
+        case .email(let email, userInfo: _):
+            return "Email address: \(email)"
```

Comment:
As far as I understand, only people speaking English would benefit from it right now. Could you sprinkle in some localization, please? 🙂

Reply:
That’s definitely a major oversight, thanks for pointing it out!

Reply:
Adding a `//TODO:` doesn't fix it :D

Reply:
Is there something else I can do now? I think when @MainasuK asked about localization earlier they said the `L10n` stuff is auto-generated so it would have to be subbed in later once the PR is merged and Crowdin runs. If there’s more I can do now I’m happy to do so tho! (I already added the corresponding keys to `app.json`)

Reply:
Ah, great, I see 👍  What I wanted to suggest is adding them to the EN `Localizable.strings`, run `swiftgen` once by hand and use the `L10n.keys` so that everything is prepared for the translated strings. Do you know what I mean?

Reply:
Looking into it now and it seems like the top-level `a11y` key here conflicts with the one defined in the stringsdict resulting in an invalid redeclaration error (https://github.com/SwiftGen/SwiftGen/issues/1017, but a different cause). I don’t want to manually update the generated file, do you have thoughts on a path forward?

Reply:
Figured out a workaround